runtime/include/blueoil_image.hpp

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2018 Leapmind Inc. */
+#ifndef RUNTIME_INCLUDE_BLUEOIL_IMAGE_HPP_
+#define RUNTIME_INCLUDE_BLUEOIL_IMAGE_HPP_
+
+#include "blueoil.hpp"
+
+namespace blueoil {
+namespace image {
+
+float *Tensor_at(Tensor &tensor, const int x, const int y); // return RGB
+Tensor Tensor_CHW_to_HWC(Tensor &tensor);
+Tensor Tensor_HWC_to_CHW(Tensor &tensor);
```

Comment:
These too seems to be used only in tests...
- Should them be a part of public API?
- If them have to be a part of public API why them are not part of the Tensor API itself?

Reply:
Currently, public API header is blueoil.hpp only.
Defined by runtime/CMakeLists.txt:file(GLOB PUBLIC_HEADERS "include/blueoil.hpp")
Tensor_CHW_to_HWC, Tensor_HWC_to_CHW in blueoil_image.hpp, so internal API at this point.

Reply:
The point was if we really need them here if they used only for testing. We could put them into some kind of helper library used for testing only except.

Reply:
I have moved CHW<=>HWC convert functions to the test folder.
https://github.com/blue-oil/blueoil/pull/9/commits/64cb5beac5a447e8803a6300ed0f4089f165750f